### DISCUSSION TITLE
Horizontal scroll with shift

### DISCUSSION THREAD

**Discussion by Cristian-farinella** [noah-nuebling/mac-mouse-fix#146] [Questions] [answered]
Am I the only one who horizontal scroll doesn't work? Before opening a bug report, I wanted to be sure. Shift and wheel by default on Mac OS horizontally scrolls, but it seems it doesn't work with smooth scrolling opened by Mouse-fix.

**Comment by noah-nuebling** [comment, marked as answer]
Hey @Cristian-farinella, I'll mark this as answered since you've moved the topic to #147. This is just to clean things up and keep the relevant discussion concentrated in one place.
